index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <meta http-equiv="x-ua-compatible" content="ie=edge">
+    
+  <!-- build:css css/main.css -->
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  <!-- endbuild -->
+
+  <title>Dangelo Arrivillaga Portfolio</title>
+</head>
+
+<body>
+  <div class="contenedor">
```

Comment:
- [ OPTIANL] please, you should consider the body as your main parent, the first child should be the header followed by the main and the footer, please, refer to the HTML and CSS best practice provided in earlier lessons